### PR TITLE
Fix(polar): polar would throw error if no label

### PR DESCRIPTION
**Pull request by susiwen8**
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [ ] new feature
- [ ] others

### What does this PR do?

Check existence of `label`

### Fixed issues

Open `test/bar-background.html`, an error will throw
Screen Shot 2021-07-08 at 11 24 28 PM

## Misc

- [ ] The API has been changed (apache/echarts-doc#xxx).
- [ ] This PR depends on ZRender changes (ecomfe/zrender#xxx).

### Related test cases or examples to use the new APIs

`test/bar-background.html`

### Merging options

- [ ] Please squash the commits into a single one when merge.

### CODE REVIEW

File: src/chart/bar/BarView.ts [hunk 1]
```diff
@@ -980,6 +980,11 @@ function updateStyle(
...
     );
 
+    const label = el.getTextContent();
+    if (!label) {
+        return;
+    }
+
     if (isPolar) {
         const position = itemModel.get(['label', 'position']);
         el.textConfig.inside = position === 'middle' ? true : null;
```
Comment (plainheart) [inline src/chart/bar/BarView.ts:990]: It seems enough to put line 990 after the `setSectorTextRotation` function, in which has set the `textConfig`.
https://github.com/apache/echarts/blob/6bd27bf3a0454abd68098ece9966ef2943543f19/src/chart/bar/BarView.ts#L990

Comment (susiwen8) [reply src/chart/bar/BarView.ts:986]: Thanks, maybe we should put  before https://github.com/apache/echarts/blob/6bd27bf3a0454abd68098ece9966ef2943543f19/src/chart/bar/BarView.ts#L989-L996 for which set label rotation.

**Review by plainheart — DISMISSED** [review]
Yes. It seems to be slightly better if we also avoid invoking `setLabelValueAnimation` when there is no label. This could reduce two unnecessary function calls: `seriesModel.getRawValue` and this function self.

**Review by Ovilia — APPROVED** [review]
LGTM

**Comment by pissang** [comment]
I found another issue that brought in https://github.com/apache/echarts/pull/15182

This `position` should be optional.
https://github.com/apache/echarts/blob/master/src/chart/bar/BarSeries.ts#L42

**Comment by susiwen8** [comment]
> I found another issue that was brought in #15182. This `position` should be optional. https://github.com/apache/echarts/blob/master/src/chart/bar/BarSeries.ts#L42
> 
> Perhaps we can fix it by hand in this PR.

@pissang Sure, done fix.

**Review by pissang — APPROVED** [review]

**Pull request merged**